packages/nextjs/utils/scaffold-eth/networks.ts

```diff
@@ -36,6 +36,11 @@ export const RPC_CHAIN_NAMES: Record<number, string> = {
   [chains.celoAlfajores.id]: "celo-alfajores",
 };
 
+export const getRpcOverrideHttpUrl = (chainId: number) => {
+  const rpcOverride = scaffoldConfig.rpcOverrides[chainId as keyof typeof scaffoldConfig.rpcOverrides];
```

Comment:
need to be changed a bit if `rpcOverrides` is optional

Reply:
Ohh maybe we do it like this: 

```ts
import { ScaffoldConfig } from "~~/scaffold.config";

const rpcOverride = (scaffoldConfig.rpcOverrides as ScaffoldConfig["rpcOverrides"])?.[chainId];
```

This will make sure we don't infer the constant value from scaffold.config eg 
`const rpcOverride: https://mainnet.buidlguidl.com`

 but 
 
 `const rpcOverride: string | undefined`

Reply:
(I also inlined it)